deck-gateway/src/main/kotlin/com/deck/gateway/event/type/UserEvents.kt

```diff
@@ -0,0 +1,16 @@
+package com.deck.gateway.event.type
+
+import com.deck.common.util.GenericId
+import com.deck.common.util.OptionalProperty
+import com.deck.common.util.UniqueId
+import com.deck.gateway.event.GatewayEvent
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+@SerialName("UserStreamsVisibilityUpdated")
```

Comment:
I think here we should add either add `@DeckExperimental` or `@DeckUnknown`, since we don't know what this event is about.